doc/configuration/Services.md

```diff
@@ -18,7 +18,7 @@ Typical services are already specified in the example configuration file.
 
 ## service_mongoose_system_metrics
 
-MongooseIM system metrics are being gathered to analyse the trends and needs of our users, improve MongooseIM, and get to know where to focus our efforts.
+MongooseIM system metrics are being gathered to analyze the trends and needs of our users, improve MongooseIM, and get to know where to focus our efforts.
```

Comment:
This is not necessary to change, both `analyse/analyze` are valid, `analyse` being the British English variant and `analize` the American English one.